modules/virt-viewing-logs-loki.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/support/virt-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-logs-loki_{context}"]
+= Viewing aggregated {VirtProductName} logs with the LokiStack
+
+You can view aggregated logs for {VirtProductName} pods and containers by using the LokiStack in the web console.
+
+.Prerequisites
+
+* You deployed the LokiStack.
```

Comment:
@apinnick  should we add a link to the Loki installation guide here?

Reply:
No. That is outside the scope of this task. In the "Additional resources" section, there is a [link to Deploying LokiStack on OCP](https://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#additional-resources_virt-troubleshootinghttps://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#additional-resources_virt-troubleshooting).